### DISCUSSION TITLE
branches

### DISCUSSION THREAD

**Discussion by plrukiurp** [community/community#19440] [Other Feature Feedback, Questions, & Ideas]
I can't delete a private branch. It's labeled as protected, and I can't remove the protection (even though I am the owner of the repository), and there are no settings that allow me to delete the branch that I made. I also tried making a new branch with the same name, but there was no option to delete the old branch, and simply said that I couldn't make the new branch. I just chose the wrong parent branch, and now I basically can't do anything.

**Comment by myConsciousness** [comment]
Just remove branch rules you created from `Settings > Branches > Branch protection rules`.

**Reply by plrukiurp** [reply]
but thats the thing i can't because the settings tab doesn't show up

**Reply by myConsciousness** [reply]
Can you share screenshots?

**Comment by Zi-Tao** [comment]
@plrukiurp
<img width="1404" alt="截圖 2022-07-02 下午3 34 34" src="https://user-images.githubusercontent.com/80264433/176991419-7ba32d15-0a13-4037-88f6-a4d370ba7fa7.png">
